lib/utils/database.rb

```diff
@@ -244,5 +244,98 @@ def self.model_to_tsv model
         end
       end
     end
+
+    def self.cleanup_whitespaces
+      DataModel::InteractionClaimType.where("type lIKE ' %' or type LIKE '% '").all.each do |t|
```

Comment:
does the L need to be capitalized?

Reply:
I don't think capitalization really matters but I pushed a commit to fix it.